scripts/gm/bin/gm

```diff
@@ -2,14 +2,17 @@
 
 set -eu
 
-# Load lib-gm either from the local folder or from the global $HOME/.gm/bin/lib-gm
+# Load lib-gm either from the local folder or from the global $HOME/.gm/bin/lib-gm or finally from an absolute path set
+# as an environment variable
 SCRIPT_0="${0:-$HOME/.gm/bin/gm}"
 export SCRIPT_DIR="${SCRIPT_0%%gm}"
 export LOCAL_LIB_GM="${SCRIPT_DIR}lib-gm"
 if [ -f "$LOCAL_LIB_GM" ]; then
   . "$LOCAL_LIB_GM"
 elif [ -f "$HOME/.gm/bin/lib-gm" ]; then
   . "$HOME/.gm/bin/lib-gm"
+elif [ -f "$LIB_GM_PATH/lib-gm" ]; then
```

Comment:
I had a hunch that the impression that HOME is always set (which might not be true) might lead to other similar expectations.

The code looks good, but if `lib-gm` is not available and `LIB_GM_PATH` is not set, it will return
```
./gm: line 14: LIB_GM_PATH: unbound variable
```
instead of the expected
```
ERROR: could not find lib-gm, exiting...
```

It's an easy fix, we just need a default for `LIB_GM_PATH`. Here's one example:
```suggestion
elif [ -f "${LIB_GM_PATH:-}/lib-gm" ]; then
```

Reply:
Oh wow, good catch. I confirmed this fix works and made the changes. In [cosmos.nix](https://github.com/informalsystems/cosmos.nix/blob/master/gm/default.nix#L7) I am running shellcheck on these scripts and I am surprised this didn't get caught. I guess it isn't always required to default vars in case they are unset? Anyways, glad you spotted this.